src/main/scala/org/broadinstitute/hail/sparkextras/OrderedRDD.scala

```diff
@@ -38,72 +44,80 @@ object OrderedRDD {
   }
 
   def apply[PK, K, V](rdd: RDD[(K, V)], fastKeys: Option[RDD[K]] = None)(implicit kOk: OrderedKey[PK, K]): OrderedRDD[PK, K, V] = {
+    val (_, orderedRDD) = coerce(rdd, fastKeys)
+    orderedRDD
+  }
+
+  def coerce[PK, K, V](rdd: RDD[(K, V)], fastKeys: Option[RDD[K]] = None)
+    (implicit kOk: OrderedKey[PK, K]): (Int, OrderedRDD[PK, K, V]) = {
```

Comment:
It seems a lot of places toss out the first element of the pair, I wonder if there should be a separate method for those places that does the throwing away itself? Keeps the API slimmer.